bindings/go/helm/Taskfile.yml

```diff
@@ -24,8 +35,102 @@ tasks:
     deps:
       - tmp
       - build
+
   install/local:
     desc: 'Installs the plugin locally to the default location'
     deps:
       - build
-    cmd: 'install $(pwd)/tmp/testdata/test-input-plugin {{.DEFAULT_CLI_PLUGIN_LOCATION}}/helm'
\ No newline at end of file
+    cmd: 'install $(pwd)/tmp/testdata/test-input-plugin {{.DEFAULT_CLI_PLUGIN_LOCATION}}/helm'
+
+  # ===== New tasks for release workflow =====
+  download/deps:
+    internal: true
+    cmd: go mod download
+
+  build:target:
+    desc: "Build Helm input plugin for a specific GOOS/GOARCH target and emit resource YAML for constructor"
+    internal: true
+    silent: false
+    prefix: '{{ .GOOS }}-{{ .GOARCH }}'
+    generates:
+      - '{{ .OUTDIR }}/bin/helminput-plugin-{{ .GOOS }}-{{ .GOARCH }}'
+      - '{{ .OUTDIR }}/resources/helminput-plugin-{{ .GOOS }}-{{ .GOARCH }}.yaml'
+    deps:
+      - tmp
+      - download/deps
+    cmds:
+      - cmd: |
+          set -euo pipefail
+          
+          # Validate required variables
+          : "{{ .GOOS }}" || (echo "GOOS is not set" && exit 1)
```

Comment:
https://taskfile.dev/docs/guide#ensuring-required-variables-are-set please check this usage instead